sbnana/CAFAna/Core/SpectrumLoader.cxx

```diff
@@ -533,6 +885,58 @@ namespace ana
       } // end for slice
     } // end for spillcut
 
+    // Truth Sigma knobs
+    // NB: We DON'T keep track of Nominal Cut/Var/etc. because we want to shift/reset shifts for the weight saving... We sacrifice potential speed here by choice.
+    for(caf::SRTrueInteractionProxy& nu: sr->mc.nu){
```

Comment:
Doesn't it make more sense to check if `fTruthNSigmasTreeDefs` has anything before we loop `nu` ? Maybe not a huge savings but thought I'd bring it up.

Reply:
Looks good!